src/wrappers.jl

```diff
@@ -180,8 +180,8 @@ for (geomtype, trait, childtype, child_trait, length_check, nesting) in (
         # Is geom a single geometry ?
         if isgeometry(geom)
             if geomtrait(geom) isa $child_trait
-                # If geom is a child_trait, then make geom a vector and call again
-                return $geomtype([geom]; extent, crs)
+                # If geom is a child_trait, then make geom a tuple and call again
+                return $geomtype((geom,); extent, crs)
```

Comment:
If it's bad for type stability, we can always revert this.  Users can choose to simply write `MultiType((geom,))` if it suits them.